google-translate-connector-product/README.md

```diff
@@ -0,0 +1,21 @@
+# google-translate-connector
+
+Say goodbye to language barriers in process automation with Axon Ivy's translation service powered by Google Translate. The connector seamlessly integrates into your existing workflows, allowing you to communicate easily regardless of the language. This marketplace item:
+
+- Supports you with an easy-to-copy demo implementation to reduce your integration effort.
+- Enables low-code citizen developers to provide multi-lingual user interfaces.
+## Demo
```

Comment:
add an empty line before the "DEMO" header. otherwise the markdown is not valid and may brake on some clients.

check out haroopad for instance:
![haroopad-rendering](https://user-images.githubusercontent.com/15085693/236754282-7f6517ec-8de2-43ed-b8dc-87e1cb5fb631.png)

Reply:
updated